share/spice/beer/beer.js

```diff
@@ -0,0 +1,31 @@
+(function(env) {
+    "use strict";
+    env.ddg_spice_beer = function(api_result) {
+
+        // Return if no book is returned
+        if(!api_result || api_result.error) {
+            return Spice.failed('beer');
+        }
+
+
+        
+        Spice.add({
+            id: "beer",
+            name: "Beer",
+            data: api_result.beers[0],
+            meta: {
+                sourceName: "Open Beer Database",
+                sourceUrl: 'http://openbeerdatabase.com/'
+            },
+            templates: {
+                group: 'base',
```

Comment:
We should be able to use the [Info](https://duck.co/duckduckhack/spice_templates_overview#info-template-group) template here, or even the Text template if we don't have much info. Seems like BreweryDB will give use lots of info that would could put into the [InfoBox](https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/tvmaze/show/tvmaze_show.js#L23)